.github/ISSUE_TEMPLATE/4-release.yml

```diff
@@ -0,0 +1,125 @@
+        -    [] Post an update in the internal release P2
+
+        #### Post-release monitoring
+
+        -    [] Follow post-release monitoring guidelines
+
+  - type: markdown
+    attributes:
+      value: |
+        ### Final release
+        
+        #### Get release artifact ready
+        
+        -    [] Validate stable tag in `readme.txt` for the release branch and update if necessary
+        -    [] Validate stable tag in `readme.txt` for the SVN `trunk` ([here](https://plugins.svn.wordpress.org/woocommerce/trunk/readme.txt))
+        -    [] Generate [changelog](https://github.com/woocommerce/woocommerce/blob/trunk/changelog.txt) ([workflow](https://github.com/woocommerce/woocommerce/actions/workflows/new-cfe-prr-pre-build-compile-changelog-file.yml))
+        -    [] Create a PR to bump version in source code with `GITHUB_TOKEN=<your_PAT_token> pnpm utils code-freeze version-bump -b release/9.9 9.9.0`
+        -    [] Review (ensure PR CI-checks are passing) and merge the PR (ensure merge CI-checks are passing)
+        -    [] Build zip file ([workflow](https://github.com/woocommerce/woocommerce/actions/workflows/build-release-zip-file.yml))
+        -    [] Validate version and stable tag in 'woocommerce.php' and `readme.txt` for the generated zip file
+
+        #### Release to WordPress.org
+
+        -    [] Notify Atomic (via P2 and Slack) before deploying release
+        -    [] Using WooRelease, perform a release dry run with e.g., `php woorelease.php wporg:release --product_version=9.9.0 --zip_file=ZIP_FILE_PATH --product_path=plugins/woocommerce https://github.com/woocommerce/woocommerce/tree/release/9.9`
+        -    [] Using WooRelease, perform release with e.g., `php woorelease.php wporg:release --product_version=9.9.0 --zip_file=ZIP_FILE_PATH --product_path=plugins/woocommerce https://github.com/woocommerce/woocommerce/tree/release/9.9 --release`
+        -    [] If the release command reports SVN-related timeouts, validate versions on https://wordpress.org/plugins/woocommerce/advanced/ and https://plugins.svn.wordpress.org/woocommerce/tags/ pages and continue if those are correct
+        -    [] Upon receiving email `[WordPress Plugin Directory] Pending release for WooCommerce`, confirm the release
+        -    [] Verify the new version is listed and downloadable on https://wordpress.org/plugins/woocommerce/ page
+        -    [] Update the Atomic comms threads about the release completion        
+
+        #### Stage and bump stable tag
+
+        -    [] For the next 4 hours, monitor the release staging: monitoring alerts (as per p1745322404770529/1745322371.115259-slack-C01DT6U03HC) and updates in the Atomic comms threads.
+        -    [] Using WooRelease, perform stable tag bump dry ran with e.g., `php woorelease.php update-stable --product_path=plugins/woocommerce https://github.com/woocommerce/woocommerce/tree/trunk 9.9.0`
```

Comment:
```suggestion
        -    [] Using WooRelease, perform the stable tag bump dry run with e.g., `php woorelease.php update-stable --product_path=plugins/woocommerce https://github.com/woocommerce/woocommerce/tree/trunk 9.9.0`
```

Reply:
Updated in 149497ec59fe9630fd9a677c7d76d6532aa59196